pkg/cmd/server/api/types.go

```diff
@@ -334,6 +334,17 @@ type AssetConfig struct {
 
 	// MasterPublicURL is how the web console can access the OpenShift api server
 	MasterPublicURL string
+
+	// ExtensionJS are file paths on the asset server files to load as scripts when the Web Console
+	// loads
+	ExtensionJS []string
```

Comment:
all fields that represent file paths need to be added by reference in `GetMasterFileReferences`, so that relative paths get automatically resolved relative to the location of the config file